client/verta/docs/tutorials/packaging/packaging.rst

```diff
@@ -14,6 +14,16 @@ ways to save a model to Verta:
 * `Using a custom model <custom_model.html>`_ to perform arbitrary computations.
 * `Using a custom model with artifact dependencies <custom_model_with_dependencies.html>`_.
 
+Registering the model
+^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This is before saving the model right?

Reply:
Hmm that makes sense, if saving model is done directly with Model Version. I think the tutorials under "Saving the model" are using experiment run, which can be done before the model registry steps though? 

Anyway, I have moved "Registering the model" section above "Saving the model".

Reply:
@convoliution What do you think about this?

Reply:
@nhatsmrt Oh, yes, 100%!
In fact this comment confused me when I first read it because I had just assumed the "Registering" section was _already_ above the "Saving" section.